addons/l10n_it_edi_sdi_coop/models/account_invoice.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding:utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import fields, models
+
+
+_logger = logging.getLogger(__name__)
+
+DEFAULT_FACTUR_ITALIAN_DATE_FORMAT = '%Y-%m-%d'
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_it_edi_transaction = fields.Char()
+    l10n_it_edi_attachment_id = fields.Many2one('ir.attachment')
```

Comment:
Do we still need this?

Reply:
Unfortunately yes. We can't return it at step_1 (otherwise account_edi will change the state from `to_send` to `sent`), but we have to return it at step_2. The only way to return the right attachment is to keep a reference to it (I'm open to any suggestion).